api.py

```diff
@@ -6,6 +6,8 @@
 from flask import Flask
 from flask import jsonify
 from flask import request
+from model import dbconnect, Species, Specimen, Genus
```

Comment:
What is this

Reply:
Hi Leah, I think you will find this is fine!